src/TRestTrackLineAnalysisProcess.cxx

```diff
@@ -0,0 +1,252 @@
+/*************************************************************************
+ * This file is part of the REST software framework.                     *
+ *                                                                       *
+ * Copyright (C) 2016 GIFNA/TREX (University of Zaragoza)                *
+ * For more information see http://gifna.unizar.es/trex                  *
+ *                                                                       *
+ * REST is free software: you can redistribute it and/or modify          *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * REST is distributed in the hope that it will be useful,               *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          *
+ * GNU General Public License for more details.                          *
+ *                                                                       *
+ * You should have a copy of the GNU General Public License along with   *
+ * REST in $REST_PATH/LICENSE.                                           *
+ * If not, see http://www.gnu.org/licenses/.                             *
+ * For the list of contributors see $REST_PATH/CREDITS.                  *
+ *************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////
+/// The TRestTrackLineAnalysis process is meant to analyze lienar tracks,
+/// such as alpha tracks. It should retreive the information after performing
+/// TRestTrackLinearization process, for the time being only works when
+/// XZ and YZ are present in the readout. Moreover, the hits have to be stored
+/// following the minimum path. It assumes that only one alpha track
+/// is present, using the parameter fTrackBalance to define the minimun energy
+/// fTrackBalance*totalEnergy of the total XZ or YZ energy to assume a single
+/// track event, if the condition is not fulfilled the event is rejected.
```

Comment:
I think we should try to do not reject events. If it is not a good event, then we will be able to discriminate it later using filters at the `analysisTree`.

Reply:
Ok, the parameted `fTrackBalance` has been removed, now we have 2 observables instead `trackBalanceX` and `trackBalanceY`